apps/web/src/components/costs/energy-costs-comparative-projection-day-card.tsx

```diff
@@ -1,11 +1,25 @@
+import { getDayComparison } from "@/components/costs/energy-projection-calculation";
 import { Card, CardContent, CardHeader, CardTitle } from "@energyleaf/ui/card";
 
-export default function EnergyCostsComparativeProjectionDay() {
+export default function EnergyCostsComparativeProjectionDay({ userData, energyData }) {
+    const predictedCost = getDayComparison(energyData, userData);
+    const color = predictedCost.absoluteDifference <= 0 ? "text-red-500" : "text-green-500";
     return (
         <Card>
             <CardHeader>
-                <CardTitle>Vergleich des hochgerechneten Tag zu letztem Tag (absolut und relativ)</CardTitle>
+                <CardTitle>Unterschied zu gestern</CardTitle>
```

Comment:
Wäre es nicht trotzdem sinnvoll, zu schreiben, dass es sich um eine Hochrechnung handelt?

Reply:
Hochrechnung steht schon darüber als Überschrift, also in page.tsx. Ist dann doppelt